packages/frontend-v2/utils/course/getCourseCoreqs.ts

```diff
@@ -19,7 +15,7 @@ export async function getRequiredCourseCoreqs(
     await Promise.all(
       coreqCourses.map(async (coreqCourse: INEUReq) => {
         if (isINEUPrereqCourse(coreqCourse)) {
-          const course = await SearchAPI.fetchCourse(
+          const course = await fetchSearchCourse(
```

Comment:
For now, we should be able to just use SearchAPI.fetchCourse after the SearchAPI is updated to not take in queries. 

Down the line, we need to think about using both the cache and batch API. Added an issue for that: #546